src/main/java/gov/usgs/wma/waterdata/groundwater/RdbWriter.java

```diff
@@ -77,10 +77,26 @@ public RdbWriter writeRow(DiscreteGroundWater dgw) {
 		}
 		writeValue(  4, time);
 
+		// THIS IS THE OLD WAY
+		/*if (StringUtils.isEmpty(dgw.levelFeetBelowLandSurface)) {
+			writeValue( 7, "");
+			writeValue( 1, "S"); // entry code for above Sea
+			writeValue( 10, dgw.verticalDatumCode);
+			writeValue( 8, dgw.levelFeetAboveVerticalDatum);
+		} else {
+			writeValue( 7, dgw.levelFeetBelowLandSurface);
+			writeValue( 1, "L"); // entry code for below Land
+			writeValue( 10, "");
+			writeValue( 8, "");
+		}*/
```

Comment:
This condition should probably be revised if we are allow the load side manage the datum. I agree with this idea of not duplicating the logic.